stamp_generation/main_stamp_generation.py

```diff
@@ -0,0 +1,74 @@
+"""
+   This script starts program for stamp stamp_generation
+
+   Once the summarizer is ready, we can move
+   this script to main.py in stampify/
+
+   Command to run:
+   /stampify/stamp_generator$ python main_stamp_generation.py
+"""
+
+import stamp_generator
+
+"""
+-Using content_type as a string for convenience for now as cannot use
+ <ContentType.FIELD: int> as a value for it in demo data.
+
+- We might need to change the content_type for each content
+  to ContentType.FIELD.name or simple "Text/Image/etc"
+  if enum can't be accessed in Jinja template in later iterations.
+"""
+
+contents_list = [{'content_type': 'TEXT',
+                  'font_style': '',
+                  'text_string': 'HTML5 Video - HTML5 Rocks',
+                  'type': 'title'},
+                 {'content_type': 'IMAGE',
+                  'img_caption': None,
+                  'img_height': 0,
+                  'img_title': None,
+                  'img_type': '',
+                  'img_url':
+                      'https://im.whatshot.in/img/2020/Mar/h-1584953735.jpg',
+                  'img_width': 0,
+                  'is_gif': False},
```

Comment:
Can we create it using methods? Some kind of factories, maybe?

Reply:
Same for stamp_pages.

Reply:
Updated! Let me know if that's what you suggested.